gridt/src/app/core/models/credentials.model.ts

```diff
@@ -0,0 +1,8 @@
+export interface Credentials {
+  username: string;
+  password: string;
+}
+
+export interface AccessToken {
+  access_token: string;
+}
```

Comment:
I would recommend referring to this file with "server-responses", as it describes both

Reply:
I did change this, but left Credentials seperate as it is not a server-response.